testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -41,7 +46,18 @@
 {
     protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
     {
-        return connectorBehavior.hasBehaviorByDefault(this::hasBehavior);
+        switch (connectorBehavior) {
+            case SUPPORTS_CREATE_SCHEMA:
+                return this.supportsCreateSchema();
+            case SUPPORTS_CREATE_TABLE:
+                return this.supportsCreateTable();
```

Comment:
incidentally ... https://github.com/trinodb/trino/pull/7274

Reply:
Nice! Looks like yours will be merged sooner than this so I'll rebase this later.